meta-iot2000-example/kas.yml

```diff
@@ -0,0 +1,23 @@
+header:
+  version: 2
+  includes:
+    - ../meta-iot2000-bsp/kas.yml
```

Comment:
Things remains under the control of the same repo, but it reads a bit nicer, indeed. Updated.